contracts/Comptroller/ComptrollerStorage.sol

```diff
@@ -70,13 +70,32 @@ contract ComptrollerV1Storage is UnitrollerAdminStorage {
         mapping(address => bool) accountMembership;
         /// @notice Whether or not this market receives XVS
         bool isVenus;
+        /**
+         * @notice Multiplier representing the collateralization after which the borrow is eligible
+         * for liquidation. For instance, 0.8 liquidate when the borrow is 80% of collateral
+         * value. Must be between 0 and collateral factor, stored as a mantissa.
+         */
+        uint256 liquidationThresholdMantissa;
+        /// @notice discount on collateral that a liquidator receives when liquidating a borrow in this market
+        uint256 liquidationIncentiveMantissa;
+        /// @notice The pool ID this market is associated with, Used to support pools/emodes
+        uint96 poolId;
+        /// @notice Flag  to restrict borrowing in certain pools/emodes.
+        bool isBorrowAllowed;
+    }
```

Comment:
AFAIU we're breaking the `markets(address)` endpoint here by adding some extra return values. Should we still expose an endpoint that returns `(bool, uint256, bool)`? Or we'd rather break it and force integrators to adjust their code?

If we choose to break `markets(address)`, do you think we can remove `isVenus` from this struct / return values of `markets`? AFAIK it's not used anywhere and is false for all markets.